nuget/CppWinrtRules.Project.xml

```diff
@@ -76,4 +76,9 @@
                 Description="Enables or disables the generation of Windows Metadata"
                 Category="General" />
 
+  <BoolProperty Name="CppWinRTEnableDefaultPrivateFalse"
```

Comment:
Since this relates to "copy local" and those terms are not in this variable consider alternatives to better communicate what this does.